web/l10n/en.json

```diff
@@ -269,6 +272,7 @@
     "eventIgnitionOff": "Ignition is OFF",
     "eventMaintenance": "Maintenance required",
     "eventTextMessage": "Text message received",
+    "eventDriverUnauthorized": "Driver Unauthorized",
```

Comment:
May be you are right. It is very unlikely when somebody that has rfid mark take not his car and put the wrong mark on the reader.